Admin/Extension/PublishTimePeriodExtension.php

```diff
@@ -41,19 +41,12 @@ public function __construct($formGroup = 'form.group_publish_workflow')
      */
     public function configureFormFields(FormMapper $formMapper)
     {
-        $dateOptions = array(
-            'placeholder' => '',
-            'required' => false,
-        );
+        $builder = $formMapper->getFormBuilder()->getFormFactory()->createBuilder(PublishTimePeriodType::class);
 
         $formMapper
-            ->with($this->formGroup, array('translation_domain' => 'CmfCoreBundle'))
-                ->add('publish_start_date', DateType::class, $dateOptions, array(
-                    'help' => 'form.help_publish_start_date',
-                ))
-                ->add('publish_end_date', DateType::class, $dateOptions, array(
-                    'help' => 'form.help_publish_end_date',
-                ))
+            ->with($this->formGroup, ['translation_domain' => 'CmfCoreBundle'])
+                ->add($builder->get('publish_start_date')) //FIXME: help message is now lost
```

Comment:
What do you think to leave the Field descriptions array even if it's not displayed to avoid loss of information?